### ISSUE TITLE
rise4fun ?

### ISSUE THREAD

**Issue by DvonHolten** [Z3Prover/z3#1842]
Folks,
maybe this isn't your playground, but since (at least) friday afternoon, the Z3-website at rise4fun does not respond.
HTH
DvH

**Comment by NikolajBjorner** [comment]
Yes, seems down. Thanks. While I contact Peli, I found this http://2.bp.blogspot.com/-KUqk5Qfj5-w/U49adMT61AI/AAAAAAAAN-E/XPsmRy6uBl8/s1600/Unknown.jpg.

**Comment by f0x52** [comment]
You can use the archived version at https://archive.fo/hca0i
